api/mesh/v1alpha1/circuit_breaker_helpers.go

```diff
@@ -0,0 +1,26 @@
+package v1alpha1
+
+import (
+	"github.com/golang/protobuf/proto"
+
+	util_proto "github.com/kumahq/kuma/pkg/util/proto"
+)
+
+func (m *CircuitBreaker) UnmarshalJSON(data []byte) error {
+	return util_proto.FromJSON(data, m)
+}
+
+func (m *CircuitBreaker) MarshalJSON() ([]byte, error) {
+	return util_proto.ToJSON(m)
+}
+func (t *CircuitBreaker) DeepCopyInto(out *CircuitBreaker) {
+	proto.Merge(out, t)
+}
+func (t *CircuitBreaker) DeepCopy() *CircuitBreaker {
+	if t == nil {
+		return nil
+	}
+	out := new(CircuitBreaker)
+	t.DeepCopyInto(out)
```

Comment:
Can you do `proto.Clone()` here?

Reply:
I would assume so. I was following the example of the controller autogen deepcopy files which always call `DeepCopyInto` from `DeepCopy`. Plan is to generate the helpers next, so makes sense to change it there instead of manually updating all these individual files.